test/e2e/feature_compatibility_version_upgrade/feature_compatibility_version_upgrade_test.go

```diff
@@ -17,7 +18,14 @@ func TestMain(m *testing.M) {
 
 func TestFeatureCompatibilityVersionUpgrade(t *testing.T) {
 	ctx := f.NewContext(t)
-	defer ctx.Cleanup()
+
+	// When the tests are ran in the evg host we don't want to clean up
+	// the context, as we need it to dump diagnostics
+	// In all other cases (e.g. tests ran locally) we still clean up
+	// the context when we finish
+	if os.Getenv("SKIP_CLEANUP") != "1" {
```

Comment:
Unless I'm missing something, we need to add this `SKIP_CLEANUP` env variable to the E2E container still right?>